[2024-02-21 06:18:39,401] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: csv_transformation_and_class_creation.run_python_script manual__2024-02-21T06:18:36.427246+00:00 [queued]>
[2024-02-21 06:18:39,408] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: csv_transformation_and_class_creation.run_python_script manual__2024-02-21T06:18:36.427246+00:00 [queued]>
[2024-02-21 06:18:39,408] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-21 06:18:39,408] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-21 06:18:39,409] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-21 06:18:39,418] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): run_python_script> on 2024-02-21 06:18:36.427246+00:00
[2024-02-21 06:18:39,420] {standard_task_runner.py:52} INFO - Started process 4472 to run task
[2024-02-21 06:18:39,424] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_transformation_and_class_creation', 'run_python_script', 'manual__2024-02-21T06:18:36.427246+00:00', '--job-id', '91', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmp16tszqj0', '--error-file', '/tmp/tmpafnnex8s']
[2024-02-21 06:18:39,424] {standard_task_runner.py:80} INFO - Job 91: Subtask run_python_script
[2024-02-21 06:18:39,460] {task_command.py:369} INFO - Running <TaskInstance: csv_transformation_and_class_creation.run_python_script manual__2024-02-21T06:18:36.427246+00:00 [running]> on host 7f99983bc7dc
[2024-02-21 06:18:39,512] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_transformation_and_class_creation
AIRFLOW_CTX_TASK_ID=run_python_script
AIRFLOW_CTX_EXECUTION_DATE=2024-02-21T06:18:36.427246+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-21T06:18:36.427246+00:00
[2024-02-21 06:18:39,597] {logging_mixin.py:115} INFO - jsjsjsjsj
[2024-02-21 06:18:39,598] {logging_mixin.py:115} INFO - Standard Output:
[2024-02-21 06:18:39,598] {logging_mixin.py:115} INFO - Standard Error:
[2024-02-21 06:18:39,598] {logging_mixin.py:115} INFO - Traceback (most recent call last):
  File "/opt/***/dags/models.py", line 1, in <module>
    from pydantic import BaseModel, HttpUrl, ValidationError,field_validator
ModuleNotFoundError: No module named 'pydantic'
[2024-02-21 06:18:39,599] {python.py:173} INFO - Done. Returned value was: None
[2024-02-21 06:18:39,611] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=csv_transformation_and_class_creation, task_id=run_python_script, execution_date=20240221T061836, start_date=20240221T061839, end_date=20240221T061839
[2024-02-21 06:18:39,647] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-21 06:18:39,669] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
